packages/vue/src/components/organisms/SfTabs/_internal/SfTab.vue

```diff
@@ -17,8 +17,13 @@
     </slot>
     <div class="sf-tabs__content">
       <div v-if="isActive" class="sf-tabs__content__tab">
-        <!--@slot Default. Here you should pass your tab content -->
-        <slot />
+        <SfScrollable v-if="maxContentHeight">
```

Comment:
@AdamPawlinski you must use data from inject like props here. 
```
<SfScrollable 
v-if="this.contentShow.maxContentHeight" 
:tab-max-content-height="this.contentShow.maxContentHeight" >
<slot/>
</SfScrollable>
```

Reply:
ok